bc_gym_planning_env/envs/synth_turn_env.py

```diff
@@ -216,7 +220,7 @@ def get_robot(self):
         return self._robot
 
 
-class RandomAisleTurnEnv(object):
+class RandomAisleTurnEnv(gym.Env):
```

Comment:
Can I inherent this class from gym.Env? Some features in the RL framework relies on that

Reply:
In short we cannot, because gym depends on `scipy`. We could take two roads: reimplement gym on our own, or try to kick out scipy from gym. We have taken the former for now.

Reply:
ok, no prob